### PR TITLE
Docs: Add notes on writing bears tutorials

### PR DESCRIPTION
**Pull request by Adrianzatreanu**
Closes https://github.com/coala-analyzer/coala/issues/2548

### CODE REVIEW

File: docs/Users/Tutorials/Linter_Bears.rst [hunk 1]
```diff
@@ -4,6 +4,12 @@ Linter Bears
 Welcome. This tutorial aims to show you how to use the ``@linter`` decorator in
 order to integrate linters in your bears.
 
+.. note::
+
+  If you are planning to create a bear that does static code analysis yourself,
```
Comment (sils1297) [inline docs/Users/Tutorials/Linter_Bears.rst]: yourself -> without wrapping a tool?

File: docs/Users/Tutorials/Writing_Bears.rst [hunk 2]
```diff
@@ -15,6 +15,11 @@ repository, go clone it with:
 All paths and commands given here are meant to be executed from the root
 directory of the *coala-tutorial* repository.
 
+.. note::
+
+    If you want to wrap an already existing linter, please refer to
```
Comment (sils1297) [inline docs/Users/Tutorials/Writing_Bears.rst]: linter -> tool

**Comment by sils1297** [comment]
otherwise good, want it :)

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2549/commits/baf83cd51cfab8f35f958c2dba2821a620e662de

**Comment by Adrianzatreanu** [comment]
reack 923645d

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2549/commits/2b4cd74858a9c4f928dac829427fa5d208156e0f and rebase please

**Comment by Adrianzatreanu** [comment]
reack 5b297ae

**Comment by sils1297** [comment]
yayyyy merge! Merge! Merge!

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**